media/css/pebbles/components/_footer.scss

```diff
@@ -43,6 +46,174 @@
         list-style-type: none;
     }
 
+    &.dark {
+        background: #000;
+        color: #fff;
+
+        a:link,
+        a:visited {
+            color: #23c7db;
+            text-decoration: none;
+        }
+
+        a:hover,
+        a:focus,
+        a:active {
+            color: #23c7db;
+            text-decoration: underline;
+        }
+    }
+
+    &.blue {
+        background-color: #005189;
+        color: #fff;
+
+        a:link,
+        a:visited {
+            color: #bee1f5;
+            text-decoration: none;
+        }
+
+        a:hover,
+        a:focus,
+        a:active {
+            color: #fff;
+            text-decoration: underline;
+        }
+    }
+
+    &.dark,
+    &.blue {
+        .logo a {
+            background-image: url('/media/img/pebbles/moz-wordmark-light-reverse.svg');
+        }
+    }
+}
+
+/* Universal redesigned site footer bug 1342235. */
+#colophon.universal {
+
+    .row {
+        @include border-box;
+        @include clearfix;
+        margin: 0 auto;
+        padding: 0 20px;
+        min-width: $width-phone;
+
+        @media #{$mq-tablet} {
+            padding: 0 10px;
+            width: $width-tablet;
+        }
+
+        @media #{$mq-desktop} {
+            width: $width-desktop;
+        }
+
+        @media #{$mq-desktop-wide} {
+            width: $width-desktop-wide;
+        }
+
+    }
+
+    .col {
+        margin-bottom: 20px;
+    }
+
+    @media #{$mq-tablet} {
+        .col {
+            @include span(4);
+            margin-bottom: 0;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .primary {
+            border-bottom: 1px solid $color-text-tertiary;
+            .col-1 {
+                @include span(2);
+            }
+            .col-2, .col-3 {
+                @include span(5);
+            }
+        }
+
+        .secondary {
+            padding-top: 20px;
+            .col-1 {
+                @include span(4);
+            }
+            .col-2 {
+                @include span(8);
+            }
+        }
+
+        .license {
+            margin-bottom: 0;
+        }
+    }
+
+    &.dark,
+    &.gray,
+    &.blue {
+        @media #{$mq-tablet} {
+            .primary {
+                border-bottom: 1px solid #fff;
```

Comment:
Totally a nitpick but it looks like the Sandstone version doesn't have this rule? The Firefox pages with a blue footer have a gray line that doesn't look as nice as the white.